client/lib/tree-select/README.md

```diff
@@ -7,13 +7,13 @@ It is called `treeSelect` because it internally uses a tree of dependencies to a
 
 `treeSelect` accepts the following arguments:
 
-* **getDependents**: A function which maps over `state` and returns all of the relevant parts of state the selector needs. You should be creating an object whose values are all the return of other selectors -- no computations allowed here.
+* **getDependents**: A function which maps over `state` and returns all of the relevant parts of state the selector needs. You should be creating an object whose values are all the returns of other selectors -- no computations allowed here.
```

Comment:
I would write: You should be creating an array whose elements are all return values of other selectors

Reply:
Perfect, done!